src/dodona/junit/JSONListener.java

```diff
@@ -1,28 +1,24 @@
 package dodona.junit;
 
-import dodona.feedback.Status;
-import dodona.feedback.StartTab;
-import dodona.feedback.StartContext;
-import dodona.feedback.StartTestcase;
-import dodona.feedback.CloseTab;
-import dodona.feedback.CloseContext;
-import dodona.feedback.CloseTestcase;
-import dodona.feedback.Message;
-import dodona.feedback.AppendMessage;
-import dodona.feedback.EscalateStatus;
+import dodona.feedback.*;
+import dodona.i18n.I18nTestDescription;
+import dodona.i18n.Language;
 import dodona.json.Json;
-
```

Comment:
Keep this newline to separate local, libs and stdlib.

Reply:
Same reason as above; my IDE cleans imports whenever I save files. I have reversed this change now.